.github/workflows/ISSUE_TEMPLATE/tck_challenge.md

```diff
@@ -0,0 +1,19 @@
+---
+name: TCK Challenge
+about: Create a TCK Challenge
+title: ''
+labels: 'challenge'
+assignees: ''
+---
+**Challenged tests**
+List the challenged tests with the fully qualified classnames and then the test methods, e.g.
+ee.jakarta.tck.faces.test.javaee7.multiFieldValidation.Spec1IT#testFailingPreconditionsNotAfterAllInputComponents
```

Comment:
can this point to something persistence specific?

Reply:
@lukasj that was an example. Feel free to suggest a change.